apps/judicial-system/api/infra/judicial-system-api.ts

```diff
@@ -0,0 +1,67 @@
+import { ref, service, ServiceBuilder } from '../../../../infra/src/dsl/dsl'
+
+export const serviceSetup = (services: {
+  backend: ServiceBuilder<'judicial-system-backend'>
+}): ServiceBuilder<'judicial-system-api'> =>
+  service('judicial-system-api')
+    .namespace('judicial-system')
+    .serviceAccount('judicial-system-api')
+    .env({
+      SAML_ENTRY_POINT: {
+        dev: 'https://innskraning.island.is/?id=judicial-system.dev',
+        staging: 'https://innskraning.island.is/?id=judicial-system.staging',
+        prod: 'https://innskraning.island.is/?id=rettarvorslugatt.prod',
+      },
+      AUTH_AUDIENCE: {
+        dev: ref((h) => `judicial-system.${h.env.domain}`),
+        staging: ref((h) => `judicial-system.${h.env.domain}`),
+        prod: 'rettarvorslugatt.island.is',
+      },
+      ALLOW_AUTH_BYPASS: { dev: 'true', staging: 'true', prod: 'false' },
+      BACKEND_URL: ref((h) => `http://${h.svc(services.backend)}`),
+      AUDIT_TRAIL_USE_GENERIC_LOGGER: 'false',
+      AUDIT_TRAIL_GROUP_NAME: 'k8s/judicial-system/audit-log',
+      AUDIT_TRAIL_REGION: 'eu-west-1',
+      XROAD_BASE_PATH_WITH_ENV: {
+        dev: 'https://securityserver.dev01.devland.is/r1/IS-DEV',
+        staging: 'https://securityserver.staging01.devland.is/r1/IS-TEST',
+        prod: 'https://securityserver.island.is/r1/IS',
+      },
+      XROAD_CLIENT_ID: {
```

Comment:
```suggestion
      CONTENTFUL_HOST: {
        dev: 'preview.contentful.com',
        staging: 'cdn.contentful.com',
        prod: 'cdn.contentful.com',
      },
      XROAD_CLIENT_ID: {
```

Reply:
A new env variable has been added since this PR was created.

Reply:
Yes, I will make sure nothing is left behind ;)